metron-platform/metron-parsing/metron-parsing-storm/pom.xml

```diff
@@ -36,6 +37,28 @@
       <artifactId>commons-lang3</artifactId>
       <version>3.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.curator</groupId>
+      <artifactId>curator-client</artifactId>
```

Comment:
Any idea why we didn't need this dependency before?

Reply:
I am not sure this is necessary anymore with the addition of [#1515](https://github.com/apache/metron/pull/1515).  I can try removing it and making sure everything still works.

Reply:
I verified this dependency is not needed.  Removed in the latest commit.